common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -204,6 +204,15 @@ public class CommonValue {
             + MODEL_MAX_LENGTH_FIELD + "\":{\"type\":\"integer\"},\""
             + ALL_CONFIG_FIELD + "\":{\"type\":\"text\"}}},\n"
             + "      \""
+            + MLModel.QUOTA_FLAG_FIELD
+            + "\" : {\"type\": \"boolean\"},\n"
+            + "      \""
+            + MLModel.RATE_LIMIT_NUMBER_FIELD
```

Comment:
no need to fix in this PR. Suggest merge `RATE_LIMIT_NUMBER_FIELD` and `RATE_LIMIT_UNIT_FIELD` into one filed